test/pass/rocm-list-split.cmake

```diff
@@ -1,5 +1,5 @@
 use_rocm_cmake()
-include(ROCMInstallTargets)
+include(ROCMUtilities)
```

Comment:
Isn't this an indication that moving that function is a potentially breaking change for rocm-cmake users?